apple/swift-docc-render
# Improve layout for landscape tutorial step assets

Bug/issue #, if applicable: 117555171

## Summary

This enhances the layout of the right-hand-side tutorial step asset column to allow for large landscape images/videos to fill the available width when possible.

## Testing

Steps:
1. Find tutorial content that has large landscape media and view it with this branch.
2. Verify that the large landscape images now fill the width of the column (with padding on either side)
3. Verify that the layout for portrait images is unchanged
4. Check for any regressions to how images/videos are presented in tutorials or other places
5. Check for any regressions to the layout/scrolling of the 2 column step/media

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
